e2e/react.test.ts

```diff
@@ -48,13 +49,52 @@ forEachCli(currentCLIName => {
         `generate @nrwl/react:lib ${libName} --publishable --no-interactive`
       );
 
-      const libTestResults = await runCLIAsync(`build ${libName}`);
+      const libTestResults = await runCLIAsync(`build ${libName} --extractCss`);
       expect(libTestResults.stdout).toContain('Bundle complete.');
 
       checkFilesExist(
         `dist/libs/${libName}/package.json`,
         `dist/libs/${libName}/index.d.ts`,
+        `dist/libs/${libName}/${libName}.esm.css`,
         `dist/libs/${libName}/${libName}.esm.js`,
+        `dist/libs/${libName}/${libName}.umd.css`,
+        `dist/libs/${libName}/${libName}.umd.js`
+      );
+    }, 120000);
+
+    it('should be able to generate a publishable react lib', async () => {
+      ensureProject();
+      const libName = uniq('lib');
+
+      runCLI(
+        `generate @nrwl/react:lib ${libName} --publishable --no-interactive`
+      );
+
+      const libTestResults = await runCLIAsync(
+        `build ${libName} --no-extract-css`
+      );
```

Comment:
It would set `extractCss` to `false` basically, because the CLI supports camelcase and snakecase flags, and the `no` in front sets the boolean to `false`. :)

Reply:
oh really? I had no idea about that "magic" feature - good to know!